duckduckgo/privacy-configuration
# Play around with moving the file structure to simplify config mainten…

…ance

This is just a draft idea, I think everything is working as expected but made it to explain the task alternative in: https://app.asana.com/0/1200890834746050/1201208544801855/f

To verify the output here:

- git checkout main
- node index.js
- cp generated generated.bk
- jq --sort-keys . generated.bk/extension-config.json > generated.bk/extension-config.json.sorted
- git checkout jkt/list-change
- node index.js
- jq --sort-keys . generated/extension-config.json > generated/extension-config.json.sorted
- diff generated.bk/extension-config.json.sorted generated/extension-config.json.sorted

Expected outcomes:

- Modified items should only be: version
- Removed features should only be where "state" is not "enabled"
